mycodo/controllers/controller_trigger.py

```diff
@@ -467,9 +480,5 @@ def receive_infrared_code_broadcast(self, code):
                                      debug=self.log_level_debug)
 
     def pre_stop(self):
-        output_channel = db_retrieve_table_daemon(OutputChannel).filter(
-            OutputChannel.unique_id == self.trigger.unique_id_3).first()
-        self.set_output_duty_cycle(
-            self.trigger.unique_id_2,
-            None,
-            output_channel=output_channel.channel)
+        self.logger.debug("Stopping")
+        self.set_output_duty_cycle(None)
```

Comment:
the db access was repeated multiple times and therefore moved into the set_output_duty_cycle method.